packages/transaction-controller/src/TransactionController.ts

```diff
@@ -950,6 +979,12 @@ export class TransactionController extends BaseController<
         transactionMeta,
         'TransactionController:confirmExternalTransaction - Add external transaction',
       );
+
+      if (transactionMeta.type === TransactionType.swap) {
+        await this.updatePostTransactionBalance(transactionMeta, {
+          numberOfAttempts: 6,
```

Comment:
Is it worth putting this in a constant such as `UPDATE_POST_TX_BALANCE_ATTEMPTS`?

Reply:
Done in https://github.com/MetaMask/core/pull/1877/commits/0750433c81974a0ffe2cb6b2ac48ac34f2051dfa